test/helpers/jsonMlMapper.ts

```diff
@@ -112,22 +125,26 @@ export function parse(jsonml: JsonML|string, document: Document): Document {
  *
  * @return  {JsonML}
  */
-export function serialize(node: Node): JsonML|string {
+export function serialize(node: Node): JsonML | string {
 	switch (node.nodeType) {
-		case node.TEXT_NODE:
```

Comment:
Much of this is prettier formatting code, but this is an actual change. It appears that the Node type only has the nodeTypes set statically.

Reply:
Ah, seems to be a slimdom issue